.github/workflows/deploy.yml

```diff
@@ -40,6 +40,7 @@ jobs:
             pip install --user --upgrade pyyaml
             pip install --user --upgrade --src=~/apps/openedx/src --editable git+https://github.com/overhangio/tutor.git@$VERSION#egg=tutor
             pip install --user --upgrade --src=~/apps/openedx/src --editable git+https://github.com/overhangio/tutor-mfe.git@$VERSION#egg=tutor-mfe
+            pip install --user --upgrade --src=~/apps/openedx/src --editable git+https://github.com/overhangio/tutor-forum.git@$VERSION#egg=tutor-forum
```

Comment:
Can you please modify this to point at your fork? That way we don't need to merge your forum pull request before we merge this one.

And its nutmeg, not nutmug 😉

Reply:
Yup, good point given also now we would double check the forum plugin before release it. 
I have renamed the branch to nutmeg, had to do a new PR :sweat_smile:.